### ISSUE TITLE
Change default loss back to MSE

### ISSUE THREAD

**Issue by shahbuland** [CarperAI/magicarp-v2#4]
BCE with logits not ideal in this instance

**Comment by shahbuland** [comment]
No longer  doing  supervised methods. No fix needed.
